docs/code-quality/ca1010.md

```diff
@@ -28,7 +28,7 @@ ms.workload:
 
 A type implements the <xref:System.Collections.IEnumerable?displayProperty=fullName> interface but does not implement the <xref:System.Collections.Generic.IEnumerable%601?displayProperty=fullName> interface, and the containing assembly targets .NET. This rule ignores types that implement <xref:System.Collections.IDictionary?displayProperty=fullName>.
 
-By default, this rule only looks at externally visible types, but this is [configurable](#configurability).
+By default, this rule only looks at externally visible types, but this is [configurable](#configurability). You can also configure additional interfaces requiring there generic pendent to be implemented.
```

Comment:
I think this needs grammatical fixes.

Reply:
@ktoliver Could you please suggest a better wording so that I can fix this part?

Reply:
"By default, this rule looks only at externally visible types, but this is configurable. You also can configure additional interfaces, which requires the interface generic pendent to be implemented."

?

Reply:
What is `generic pendent`? I am not sure I understand...

Reply:
Google translate isn't my friend... I meant something like the `generic match`: `IEnumerable` -> `IEnumerable<T>`.

Reply:
Probably I'd say "generic equivalent".